<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>830</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>570</width>
    <height>830</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>570</width>
    <height>830</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Core Generating Network</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Operation</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_critical_variable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Critical Variable not set</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_no_of_iterations">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>230</y>
      <width>250</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>250</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_no_of_threads">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>200</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>90</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>No of Threads</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_status">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>550</width>
      <height>350</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>550</width>
      <height>350</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>550</width>
      <height>200</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_select_dataset">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>50</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Dataset</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_initial_population">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>113</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Initial Population Size</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_no_of_threads">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>130</y>
      <width>250</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>250</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_set_critical_variable">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>180</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Critical Variable</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_no_of_iterations">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>82</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>No of iterations</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_mutation_percentage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Mutation Percentage</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_start">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>380</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Training</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_initial_population">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>280</y>
      <width>250</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>250</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>32</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Status</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_mutation_percentage">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>330</y>
      <width>250</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>250</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>30</height>
     </size>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::NoWrap</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clear_critical_variable">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>180</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Critical Variable</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_select_network">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>90</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Network</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_select_dataset">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>440</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_select_network">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>440</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>570</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
